### PR TITLE
Changelogs for Digital Twin Manager 1020.1.12

### PR DESCRIPTION
**Pull request by lra-sag**
This PR contains the changelogs for Digital Twin Manager 1020.1.12

### CODE REVIEW

**Comment by lra-sag** [comment]
Closing the current one so that new one is created with updated info

File: content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Upgraded angular and webSDK versions  [skip-cl]"
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md:3]:
```suggestion
title: "Upgraded Angular and webSDK versions  [skip-cl]"
```

Comment (BeateRixen) [reply content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md:3]: What does [skip-cl] mean here?

File: content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1309"
+version: "1020.1.12"
+---
+For DTM, the webSDK version and angular version has been upgraded to 1020.0.22 and 17.3.1 respectively.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md]:
```suggestion
For the Digital Twin Manager application, the Web SDK version and the Angular version have been upgraded to 1020.0.22 and 17.3.1 respectively.
```

File: content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md (hunk 1 above)
Comment (lra-sag) [reply content/change-logs/application-enablement/dtm-1020.1.12-upgraded-angular-and-web-sdk-versions-skip-cl.md]: This is an internal identifier to not add content to the changelogs. The PR is now updated.

**Review by rahultsag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**